extra/mender-device-docker/Dockerfile

```diff
@@ -0,0 +1,61 @@
+# Creates a container which acts as a bare bones non-VM based Mender
+# installation, for use in tests.
+FROM ubuntu:20.04 AS build
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt update && apt install -y make git build-essential golang liblzma-dev \
+    jq libssl-dev libglib2.0-dev
+
+ARG MENDER_CLIENT_VERSION=master
+ARG MENDER_CONNECT_VERSION=master
+ARG MENDER_CONFIGURE_VERSION=master
```

Comment:
@kacf This one is challenging in combination with the build job I am creating in `mender-qa`.

As you know, mender-configure will use versions like "latest", which then we need to translate here to master. Assuming that this is going to be a very stable repo, I am thinking of not taking this as an argument and just add logic in this Dockerfile to checkout latest git tag (or master, if no tags).

This will be correct most of the times, and we could still rework later if we really need to pass a version from mender-qa to here. What do you think?

Reply:
I reworked following my above suggestion. Let me know if you disagree.

Reply:
Sorry, a bit late to the party, but yes, I agree, better to leave it out if we can.